pkg/controller/operators/catalog/operator_test.go

```diff
@@ -464,62 +463,67 @@ func fakeConfigMapData() map[string]string {
 	return data
 }
 
-// NewFakeOperator creates a new operator using fake clients
-func NewFakeOperator(clientObjs []runtime.Object, k8sObjs []runtime.Object, extObjs []runtime.Object, regObjs []runtime.Object, namespace string, stopCh <-chan struct{}) (*Operator, []cache.InformerSynced, error) {
+// NewFakeOperator creates a new operator using fake clients.
+func NewFakeOperator(clientObjs []runtime.Object, k8sObjs []runtime.Object, extObjs []runtime.Object, regObjs []runtime.Object, operatorNamespace string, namespaces []string, stopCh <-chan struct{}) (*Operator, error) {
```

Comment:
the `namespace` to `operatorNamespace` change is confusing - in other parts of the tests we use `namespace` to refer to the namespace that OLM is running in and `operatorNamespace` to refer to the namespace that OLM is watching. 

i.e. if we change it here we should change it everywhere to be consistent